### PR TITLE
Feature/2017 09 29 percona live 2017

### PR DESCRIPTION
**Pull request by ChrisDeBruyne**

### CODE REVIEW

File: _posts/2017-09-29-perconaLive2017.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+>Beside enjoying the local brews and drafts we attended several sessions out of which we highlight some sessions in this blog post.
+
+## MongoDB Shootout: MongoDB Atlas, Azure CosmosDB and Doing It Yourself
+
+When running MongoDB in the cloud, you have several options. Here David Murphy compared [MongoDB Atlas](https://www.mongodb.com/cloud/atlas){:target="_blank"}, [CosmosDB](https://docs.microsoft.com/en-us/azure/cosmos-db/introduction){:target="_blank"} and good old DIY.
+
+Using Atlas, you get monitoring, automation and the possibility to pay for backups.
+You pay per instance and you have a wide variety of instances and regions to choose from on AWS, GCP and Azure.
+Biggest downside is that the cost is about 44% more than running you own servers in the cloud.
+This means that, in contrast to DIY, you continuously pay more instead of writing of your initial investment and paying less in the end.
```
Comment (Turbots) [inline _posts/2017-09-29-perconaLive2017.md]: **writing off** (double F)

File: _posts/2017-09-29-perconaLive2017.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+The monitoring is really good but the problem here is that if you have have a polyglot environment with your own monitoring and alerting, you can not integrate it with Atlas so you end up with yet another tool.
+Upgrading is really easy and just a click of the button thanks to the automation.
+Backups need to be payed for per GB and are taken continuously.
```
Comment (Turbots) [inline _posts/2017-09-29-perconaLive2017.md]: paid

File: _posts/2017-09-29-perconaLive2017.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+CosmosDB is offered by Microsoft on Azure and claims to be MongoDB compatible. 
+This is not completely true because they have no support for the aggregation framework.
+So you can only use it for simple CRUD operations.
+The pricing is based on a pay-per-operation model which means it's really hard to figure out what your cost will be and how it will evolve over time.
+Here you also have the downside of continuous paying more than DIY.
```
Comment (Turbots) [inline _posts/2017-09-29-perconaLive2017.md]: continuously?

File: _posts/2017-09-29-perconaLive2017.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+You choose what cloud or hardware.
+You choose what you want to monitor and how you alert.
+You choose how often and when you backup.
+But of course, it's all up to you.
+
+
+## Visualize Your Data With Grafana
+
+[Grafana](https://grafana.com/){:target="_blank"} is used to build monitoring dashboards based on time series data.
+Grafana supports a wide range of [data sources](https://grafana.com/plugins?type=datasource){:target="_blank"} to get it's data and generate the dashboard.
```
Comment (Turbots) [inline _posts/2017-09-29-perconaLive2017.md]: **its** data

File: _posts/2017-09-29-perconaLive2017.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+There are already a lot of pre-build [dashboards](https://grafana.com/dashboards){:target="_blank"} you can use and customize for your own needs.
```
Comment (Turbots) [inline _posts/2017-09-29-perconaLive2017.md]: pre-built (met T)

File: _posts/2017-09-29-perconaLive2017.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+This is probably the most stunning talk of the conference.
+OtterTune is a tool developed by students and researchers at Carnegie Mellon to automatically tune your database.
+This is done by making clever use of previously collected data of other tunings and applying machine learning to it.
+The presented results showed that for the given workload OtterTune was at par with DBA's which had double digit years of experience.
+Looking at this from the bright side, OtterTune would help DBA's to focus on other areas then figuring out which combination of the multiple settings they should use to tune their database.
+It would definitely help to do better Database Reliability Engineering.  
+
+# Conclusion
+
+This is just a small portion of the huge amount of sessions at Percona Live, but off course, one needs to choose.
```
Comment (Turbots) [inline _posts/2017-09-29-perconaLive2017.md]: of course

**Comment by yannickdeturck** [comment]
Still need to rename the file to have the correct publication date and fix the blogpost image (incorrect size, see index page and compare it to the blogpost right next to it).

**Pull request merged**